platforms/software/ivy/src/main/java/org/gradle/api/publish/ivy/tasks/PublishToIvyRepository.java

```diff
@@ -87,31 +87,20 @@ public PublishToIvyRepository() {
     /**
      * The publication to be published.
      *
-     * @return The publication to be published
+     * Currently only instances of IvyPublication are supported.
      */
     @Internal
-    @ToBeReplacedByLazyProperty
-    public IvyPublication getPublication() {
+    @ReplacesEagerProperty
+    public Property<IvyPublication> getPublication() {
```

Comment:
❓ Do we want to have this as `Property`? It also looks like a nested property
cc @jvandort